pos_frequency_pipeline.py

```diff
@@ -1,8 +1,37 @@
 """
 Implementation of POSFrequencyPipeline for score ten only.
 """
+import re
+
+from pathlib import Path
+
+from visualizer import visualize
+
+from constants import ASSETS_PATH
 
 
 class POSFrequencyPipeline:
-    def __init__(self, assets):
-        pass
+    def __init__(self, corpus_manager):
+        self.corpus = corpus_manager
+
+    def run(self):
+        frequencies = self._count_frequencies()
+        path = Path(ASSETS_PATH) / 'pos_frequencies.png'
```

Comment:
read the requirements for the pipeline one more time: there should be images with names N_image.png and not a single image about frequencies in all texts